test/hotspot/jtreg/gc/metaspace/TestMetaspacePerfCounters.java

```diff
@@ -244,15 +244,17 @@ private static void checkPerfCounters(String ns) throws Exception {
 
         final int MaxAttempts = 10;
```

Comment:
This should be used or removed as well, right?

Reply:
`MaxAttempts` is used in the code. I prefer to have a named constant than a raw embedded value almost every time, also this time, but I can be convinced to inline it as well. Declaring it in the for-loop only clutters the loop body.

Reply:
Yes, sorry, I do not know what I was thinking... Looks good to me.

Reply:
And thank you for refactoring the loop!